test/integration/ddl_sharding_key_test.lua

```diff
@@ -0,0 +1,178 @@
+local fio = require('fio')
+
+local t = require('luatest')
+local crud = require('crud')
+
+local helpers = require('test.helper')
+
+local ok = pcall(require, 'ddl')
+if not ok then
+    t.skip('Lua module ddl is required to run test')
+end
```

Comment:
Can we enforce it on CI? Say, fail instead of skipping if `os.getenv('CI') ~= nil`? (The variable is from GitHub Actions [docs](https://docs.github.com/en/actions/reference/environment-variables).)

Reply:
I don't like it, it make testing more complex and dependent on external environment variables.
Let's replace it to fail if you don't like silent skip.